application/performance/locust/scenario2.py

```diff
@@ -0,0 +1,50 @@
+import json
+from locust import HttpUser, task, LoadTestShape, tag
+import config as config
+
+
+class LoadFile:
+    def __init__(self):
+        self.file = "ccs_5_destinations.json"
+
+    def set_file(self, file: str):
+        self.file = file
+
+    def get_file(self):
+        return self.file
+
+
+current_file = LoadFile()
+
+
+class FiveDest(HttpUser):
+    weight = 1
+
+    def on_start(self):
+        with open(f'{config.ccs_prefix}{current_file.get_file()}') as json_file:
```

Comment:
This seems to be different to how it's done elsewhere, but as before this could all be inherited as common code as it does the same thing